### ISSUE TITLE
Timezone support for logging [feat]

### ISSUE THREAD

**Issue by chrish987** [zwave-js/zwavejs2mqtt#490]
Is it possible to add TZ support to the docker image, and then have logs timestamped in local timezone?

I can install this manually or build a container from the published images - but nice to have it in by default. I have tested the manual install and the log date is then in the correct TZ.

Be nice to be able to pass the TZ in a docker ENV variable.

**Comment by robertsLando** [comment]
@chrish987 Isn't it already there? 

https://github.com/zwave-js/zwavejs2mqtt/blob/master/docker/Dockerfile#L42

**Comment by billiaz** [comment]
@robertsLando Generally would be nice if the logs had an ISO8601 timestamp, probably from the following only TZ missing

`2021-02-07 17:39:43.081 INFO ZWAVE: Node 35:`

**Comment by robertsLando** [comment]
AFAIK logs are always in UTC format

**Comment by chrish987** [comment]
this is working ok for me. If I set the TZ=Pacific/Auckland env variable in the docker-compose.yaml the logs are in the correct TZ. No TZ piece to the string - but the time is right. Sorry I did not test it very well before I opened the issue.

2021-02-09 17:33:08.557 INFO ZWAVE: Node 22: value updated: 112-0-8 5 => 5
2021-02-09 17:33:08.742 INFO ZWAVE: Node 6: value updated: 112-0-103 0 => 0
2021-02-09 17:33:08.963 INFO ZWAVE: Node 22: value updated: 112-0-9 1 => 1
2021-02-09 17:33:09.155 INFO ZWAVE: Node 6: value updated: 112-0-111 3 => 3
2021-02-09 17:33:09.376 INFO ZWAVE: Node 22: value updated: 112-0-10 0 => 0
